### PR TITLE
remove commas from decimal places in addCommas module

### LINKED ISSUE #59: در نظر گرفتن اعشار در کاما
https://github.com/persian-tools/persian-tools/blob/35422fc5bf6f85642dc1c5ffc71d1ebe31d9bafa/test/addCommas.spec.ts#L6

<p dir="rtl">
بنظرم بهتره در تست ها اعشار در کاما در نظر گرفته شود<br>
مثال :  30,000,000.02
</p>

### PR DESCRIPTION
**Pull request by mediv0**
fixing #59

### CODE REVIEW

**Comment by moh3n9595** [comment]
I think that it's not the right fit for adding commas to floating-point numbers. 
For example changing `30000000.12564600024646` to something like `30,000,000.12,564,600,024,646` doesnt make any sense.

**Comment by mediv0** [comment]
> I think that it's not the right fit for adding commas to floating-point numbers.
> For example changing `30000000.12564600024646` to something like `30,000,000.12,564,600,024,646` doesnt make any sense.

current RP just adds more tests for more coverage not changing the actual implementation. but I agree with you adding commas to floating point is nonsense.

We have to keep it as it is
`12500.9999`  to  `12,500.9999`

but current implementation adds commas to floating numbers

as we see here
```js
 Expected: "12,500.99999"
 Received: "12,500.99,999"
```

**Comment by mediv0** [comment]
9c360c3 will remove commas from decimal places

```js
 Expected: "12,500.99999"
 Received: "12,500.99999"
```

I had to use `Negative lookbehind (?<!\.\d*)` and this expression is not supported on safari and IE11

**Comment by ali-master** [comment]
> not supported on safari and IE11

I think we have to split the input by dot and then add commas to the first section and then merge them together. by this situation, we supports all other browsers.

Reference: https://caniuse.com/js-regexp-lookbehind

File: src/modules/commas/add.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ const addCommas = (number?: number | string): string | undefined => {
 		? (digitsFaToEn(convertedToString) as string)
 		: convertedToString;
 
-	return tokenizedToEnglish.replace(/(\d)(?=(\d{3})+(?!\d))/g, "$1,");
+	return tokenizedToEnglish.replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ",");
```
Comment (ali-master) [inline src/modules/commas/add.ts]: image
I think we have to split the input by dot and then add commas to the first section and then merge them together. by this situation, we supports all other browsers.

**Review by ali-master — CHANGES_REQUESTED** [review]
Support other browsers

**Pull request merged**